torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -1179,6 +1175,36 @@ def call_triton_kernel(
                 "Please use a Config in @triton.autotune instead."
             )
 
+        # Run prune_configs_by to filter the configs
+        if isinstance(variable.kernel, Autotuner):
```

Comment:
Can we check that either of the config pruning callbacks is actually set before we do the below?